### DISCUSSION TITLE
Calibration multi-objective

### DISCUSSION THREAD

**Discussion by fmanquehual** [mhm-ufz/mHM#105] [General]
Hi!

I'm interested in calibrating the model using the option 18 in mHM v5.13.2, this is:

!> (18) (reserved) please use the next number when implementing a new one
!>      MO: Q:  1st objective: 1.0 - lnNSE(Q_highflow)  (95% percentile)
!>          Q:  2nd objective: 1.0 - lnNSE(Q_lowflow)   (5% of data range)

When i select that option, the messages is the following:

  Run mHM optimization
***ERROR: mhm_driver: The given objective function number 18 in mhm.nml is not valid!
STOP 1

The title of the option says "reserved", so how can I use it? Or how can I implemented? Because in the namelist when I select the option says: 

!> further functions can be implemented in mo_objective_function and mo_mrm_objective_function
opti_function = 18

Thanks!

**Comment by StephanThober** [comment]
Hi Fmanquehaul,

thank you for using mHM. This objective functions is currently not available for use. If you are looking for an objective function that combines both high and low flows, then you could also use objective function 5. 

We are currently not providing a hands-on for adding new objective functions, but we will discuss this among the developers.

Kind regards,
Stephan

**Reply by fmanquehual** [reply]
Thank you so much Stephan.
